object-search/README.md

```diff
@@ -0,0 +1,253 @@
+
+Features that come out of the box:
+
+- Interactive query
+- Parallel replicas
+- Index with shards
+- Containerization
+- REST and gRPC gateway
+- Dashboard monitor
+
+To save you from dependency hell, we'll use the containerized version in these instructions. That means you only need to have [Docker installed](https://docs.docker.com/get-docker/). No Python virtualenv, no Python package (un)install. 
+
+The code can of course run natively on your local machine, please [read the Jina installation guide for details](https://docs.jina.ai/chapters/install/via-pip.html).
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+- [TL;DR: Just Show Me the Pokemon!](#tldr-just-show-me-the-pokemon)
+- [Download and Extract Data](#download-and-extract-data)
+- [Index Image Data](#index-image-data)
+- [Query Top-K Visually Similar Images](#query-top-k-visually-similar-images)
+- [Build Docker Image](#build-docker-image)
+- [Troubleshooting](#troubleshooting)
+- [Documentation](#documentation)
+- [Community](#community)
+- [License](#license)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+
+## TL;DR: Just Show Me the Pokemon!
+
+> *I want Pokémon! I don't care about Jina cloud-native neural search or whatever big names you throw around, just show me the Pokémon!*
+
+We have a pre-built Docker image ready to use:
+
+```bash
+docker run -p 34567:34567 -e "JINA_PORT=34567" jinaai/hub.app.bitsearch-pokedex search
+```
+
+Then you can `curl`/query/js it via HTTP POST request. [Details here](#query-via-rest-api).
+
+## Download and Extract Data
+### Use Flickr8k
+Although the model is trained on Flickr30k, you can test on Flickr8k dataset, which is a much smaller version of 
+flickr30k.
+
+To do so, instead of downloading the flickr30k from kaggle, just take its 8k counterpart
+
+```bash
+kaggle datasets download adityajn105/flickr8k
+unzip flickr8k.zip 
+rm flickr8k.zip
+mv Images data/f8k/images
+```
+
+## Index Image Data
+
+```bash
+docker run -v "$(pwd)/data:/data" -v "$(pwd)/workspace:/workspace" -e "JINA_LOG_PROFILING=1" -p 5000:5000 jinaai/hub.app.objectsearch index
+```
+
+#### Command Line Arguments Explained
+- `$(pwd)/data`: the directory where all your images are stored (jpg/png are supported). You can change it to any path you like, just make sure it's an absolute path
+- `$(pwd)/workspace`: the directory where Jina stores indexes and other artifacts. 
+- `"JINA_LOG_PROFILING=1" -p 5000:5000`: optionally enables dashboard monitoring.
+
+### Behind the Scenes
+
+<table>
+<tr>
+<td> Python API </td>
+<td> index.yml</td>
+<td> <a href="https://github.com/jina-ai/dashboard">Flow in Dashboard</a></td>
+</tr>
+<tr>
+<td> 
+
+```python
+from jina.flow import Flow
+
+f = Flow.load_config('flow-index.yml')
+
+with f:
+    f.index(input_fn, batch_size=128)
+```
+
+</td>
+<td>
+  <sub>
+
+```yaml
```

Comment:
Either make sure to align this with the actual Flows, or remove it and have a smaller README.md